pkg/authz/policy_test.go

```diff
@@ -38,9 +39,10 @@ func TestDeniesWhenPolicySaysDeny(t *testing.T) {
 	result, err := authorizer.Authorize(request)
 	require.NoError(t, err)
 	require.False(t, result.Approved)
+	require.True(t, result.TokenValid)
 }
 
-func TestPolicyEvaluatedAgainstHTTPRequest(t *testing.T) {
+func TestPolicyEvaluatedAgainstGoodHTTPRequest(t *testing.T) {
```

Comment:
It would be good to see a test that triggers the error from NewPolicyAuthorizer (if straight-forward).  Not critical though.

Reply:
It's not super easy, basically only some kind of mangled input structure or mangled policy can cause it. I've simplified the error condition so hopefully it's even less critical.